timeseries/src/autogluon/timeseries/learner.py

```diff
@@ -223,7 +223,5 @@ def get_info(self, include_model_info: bool = False, **kwargs) -> Dict[str, Any]
         learner_info.pop("random_state", None)
         return learner_info
 
-    def refit_full(self, models="all"):
-        # TODO: Implement refitting
-        # return self.load_trainer().refit_full(models=models)
-        raise NotImplementedError("refitting logic currently not implemented in autogluon.timeseries")
+    def refit_full(self, model: str = "best") -> Dict[str, str]:
```

Comment:
Any reason why we change the model from `all` to `best`?

Reply:
Thanks, changed this to `"all"` to align with Tabular.